README.md

```diff
@@ -369,46 +370,46 @@ The following 44 metrics are implemented in PyKEEN.
 
 | Name                                                                                                                                           | Interval                    | Direction   | Description                                                             | Type           |
 |------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------|-------------|-------------------------------------------------------------------------|----------------|
-| [Accuracy](https://en.wikipedia.org/wiki/Evaluation_of_binary_classifiers#Single_metrics)                                                      | $[0, 1]$                    | 📈           | The ratio of the number of correct classifications to the total number. | Classification |
```

Comment:
This change looks unrelated

Reply:
the thing I haven't been able to figure out about this is why locally the `pykeen readme` command changes this, but on CI, it does not expect it